config/v2/exithandler/controller/kustomization.yaml

```diff
@@ -1,3 +1,5 @@
+namePrefix: data-science-pipelines-operator-
```

Comment:
This should not be part of this PR but part of a follow-up issue to fix later

Reply:
Resolving all these conversations as I took @HumairAK 's suggestion of adding these configmaps and secrets to the makev2deploy kustomize build which I missed to notice in the first glance.